README.md

```diff
@@ -38,6 +38,7 @@ you are interested in battle-tested, production-ready Terraform code, check out
 1. `cd` into the module's folder (e.g. `cd non-prod/us-east-1/qa/mysql`). 
 1. Note: if you're deploying the MySQL DB, you'll need to configure your DB password as an environment variable:
    `export TF_VAR_master_password=(...)`.
+1. Run `terragrunt init` to initialize provider plugins.
```

Comment:
They latest version of Terragrunt should do this automatically if this is the first time you checked out a module. However, if you add a module or provider, or change its version, then you do have to re-run `init` manually.

Are you not seeing this behavior?

Reply:
I'm using `v0.14.7` installed with homebrew and I confirm I don't see this behaviour, I have to manually trigger the `init` command even if this is the first time I check out a module.

Reply:
Could you file a bug in the Terragrunt repo with the full log output? It just sounds like a bug we need to fix!

Reply:
I will try to do it.